conan/tools/microsoft/visual.py

```diff
@@ -49,13 +56,14 @@ def _write_conanvcvars(conanfile, auto_activate=True):
             minor = version_components[1]
             # The equivalent of compiler 19.26 is toolset 14.26
             vcvars_ver = "14.{}".format(minor)
-        cvars = vcvars_command(vs_version, architecture=vcvarsarch, platform_type=None,
-                               winsdk_version=None, vcvars_ver=vcvars_ver)
-    if cvars:
+        intel_vars_setup_command = vcvars_command(vs_version, architecture=vcvarsarch,
```

Comment:
Using the ``intel`` name here is very confusing, this one is not related to intel, right?